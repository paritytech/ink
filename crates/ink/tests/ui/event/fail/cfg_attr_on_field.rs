#[ink::event]
pub struct Event {
    #[cfg(feature = "std")]
    pub a: [u8; 32],
}

fn main() {}