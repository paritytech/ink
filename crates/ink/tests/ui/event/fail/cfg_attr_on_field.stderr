error: conditional compilation is not allowed for event fields
 --> tests/ui/event/fail/cfg_attr_on_field.rs:3:5
  |
3 |     #[cfg(feature = "std")]
  |     ^
